dll/win32/netid/lang/zh-TW.rc

```diff
@@ -65,23 +65,23 @@ STRINGTABLE
 BEGIN
     1 "* 未知 *"
     2 "WORKGROUP"
-    3 "嘗試讀取域成員身份資訊時出現以下錯誤:"
+    3 "嘗試讀取網域成員身份資訊時出現以下錯誤:"
     4 "電腦名稱更改"
     5 "工作群組:"
-    6 "域:"
-    22 "歡迎到 %1 工作群組。"
-    23 "歡迎到域 %1。"
+    6 "網域:"
+    22 "歡迎加入 %1 工作群組。"
+    23 "歡迎加入 %1  網域。"
```

Comment:
seems there are two spaces after ```%1```. is this needed?

Reply:
oh, there's only need one space. my mistake.